ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -92,41 +95,134 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(__unused RCTPromiseRejectBlock)reject)
 {
-  UILocalNotification *notification = [self _localNotificationFromPayload:payload];
+  UNMutableNotificationContent* content = [self _localNotificationFromPayload:payload];
 
   [EXUtil performSynchronouslyOnMainThread:^{
-    [RCTSharedApplication() presentLocalNotificationNow:notification];
+    UNNotificationRequest* request = [UNNotificationRequest requestWithIdentifier:content.userInfo[@"id"]
+                                                                          content:content
+                                                                          trigger:nil];
+    [[EXUserNotificationCenter sharedInstance] addNotificationRequest:request withCompletionHandler:^(NSError * _Nullable error) {
+      if (error != nil) {
+        reject(@"E_NOTIF", [NSString stringWithFormat:@"Could not add a notification request: %@", error.localizedDescription], error);
+      } else {
+        resolve(content.userInfo[@"id"]);
+      }
+    }];
   }];
 
-  resolve(notification.userInfo[@"id"]);
+}
+
+RCT_EXPORT_METHOD(addCategory: (NSString *)categoryId
```

Comment:
name of the method exported to the user should match name of the native method; append `Async`